flow/connectors/utils/heartbeat.go

```diff
@@ -37,12 +36,8 @@ func HeartbeatRoutine(
 
 // if the functions are being called outside the context of a Temporal workflow,
 // activity.RecordHeartbeat panics, this is a bandaid for that.
```

Comment:
outdated comment

Reply:
Not quite outdated. This function is to avoid panic from `activity.RecordHeartbeat` still